### PR TITLE
docs(exp): add pause at TOD

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary
Supports A32NX PR for exp testing:
- https://github.com/flybywiresim/a32nx/pull/7165

Additionally makes note that the flyPadOS 3 is now released in Development.

### Location
- docs/fbw-a32nx/support/exp.md

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

**Comment by 2hwk** [comment]
Add known Sim Limitation:

Note that while aircraft simulation is suspended (i.e. fuel consumption, airspeed, altitude, physics), environmental simulation will continue (Notably Live Weather, Live Traffic/AI Traffic). Consequences of this include that TCAS TA/RAs will still be issued and if there is a significant change in the weather over time (say a few hours), your vertical descent profile and performance figures may no longer be accurate, which means that the actual T/D point may now have shifted from the originally calculated value. 

The timestamp issued in the pause pop-up shows your real local system's time (localized to your locale) to help mitigate against this problem. 

This is considered a sim limitation as the pause event we utilize does not affect the MSFS world environment. Along this line of thinking, we speculate that without historical live weather and traffic, it would not possible to inject or restore the state that the world was in when the pause was initiated (the time slider in the weather toolbar only affects lighting and will still reflect live wind and weather data). 

Flying using a weather preset and without AI/live traffic avoids this limitation entirely, but would also be a compromise in other ways.

**Comment by Valastiri** [comment]
Waiting for multiple EXP PRs to be pushed to development for A32NX

File: docs/fbw-a32nx/support/exp.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ thread to report any issues - [How to Report Issues](#how-to-report-issues).
...
 
+### Pause at Top of Descent (TOD)
+
+- New setting in located in the EFB under Realism - Pause at TOD (configurable by distance between 0-50 nm before TOD)
+- When enabled, flight will pause at the specified distance before TOD
+- If the TOD point shifts before your present position, or AP mode reverts in CRZ, this will pause the simulation.
+
 ### Hydraulic Gear System
 
 Featured released in Development Version. See our guide for usage and known issues.
```
Comment (Benjozork) [inline docs/fbw-a32nx/support/exp.md]:
```suggestion
Feature released in Development Version. See our guide for usage and known issues.
```

File: docs/fbw-a32nx/support/exp.md [hunk 2]
```diff
@@ -45,36 +51,9 @@ These features are not yet available but will be implemented at a later time.
...
-- Presets for customizable lighting settings and predefined aircraft states
-- Improved settings (better structure and more configuration options)
-- Onscreen keyboard
-- Tooltips
-- Themes
-    - Blue
-    - Dark
-    - Light
-- Multilingual - English, French, Spanish, German, Russian - more to come....
+Featured released in Development Version. See our guide for usage and known issues.
```
Comment (Benjozork) [inline docs/fbw-a32nx/support/exp.md]:
```suggestion
Feature released in Development Version. See our guide for usage and known issues.
```

**Review by Benjozork — CHANGES_REQUESTED** [review]

**Comment by Valastiri** [comment]
added sim limitation as admonition

**Pull request merged**